src/components/AppSidebar/AppSidebar.vue

```diff
@@ -181,19 +175,17 @@ include a standard-header like it's used by the files app.
 						<!-- favourite icon -->
 						<div v-if="canStar || $slots['tertiary-actions']" class="app-sidebar-header__tertiary-actions">
 							<slot name="tertiary-actions">
-								<a v-if="canStar"
-									class="app-sidebar-header__star"
+								<Button v-if="canStar"
+									type="tertiary-no-background"
 									@click.prevent="toggleStarred">
-									<span v-if="starLoading" class="icon-loading-small" />
-									<Star v-else
-										:class="{
-											'star--starred': isStarred,
-											'star--star': !isStarred,
-										}"
-										class="star"
-										:size="20"
-										decorative />
-								</a>
+									<template #icon>
+										<span v-if="starLoading" class="icon-loading-small" />
+										<Star v-else
+											:fill-color="isStarred ? '#FC0' : 'currentColor'"
```

Comment:
would be great, if you could use scss variables instead of color code (#FC0) - this is better to maintain

Reply:
could you please use :title="t('xx', 'xx')" for the description of the icon?

Reply:
> would be great, if you could use scss variables instead of color code (#FC0) - this is better to maintain

From a ripgrep it looks like this is the only component that uses this so I wouldn't want to inflate the variables file with a variable that is only used in one place until it is used elsewhere

> could you please use :title="t('xx', 'xx')" for the description of the icon?

Aria labels have been added to the buttons instead

Reply:
Looks like it already exist: https://github.com/nextcloud/server/blob/master/core/css/variables.scss#L70= but not sure if it is easy to use in your case.

Reply:
Available in server scss variables but not https://github.com/nextcloud/nextcloud-vue/blob/2ac55083188ff5ebb4a3defee052f2508b4b165e/src/assets/variables.scss so I think we should keep it as local as possible and not add it when it's only used once in the vue component library